app/src/main/java/org/fossasia/phimpme/editor/fragment/RecyclerMenuFragment.java

```diff
@@ -205,15 +219,35 @@ public void onBindViewHolder(mRecyclerAdapter.mViewHolder holder, final int posi
             layoutParams.gravity = Gravity.CENTER_HORIZONTAL;
             holder.icon.setLayoutParams(layoutParams);
             holder.title.setText(titlelist.getString(position));
+            holder.wrapper.setBackgroundColor(Color.TRANSPARENT);
+
+            if(currentSelection == position)
+                holder.wrapper.setBackgroundColor(ContextCompat.getColor(getContext(), R.color.md_grey_200));
 
             holder.itemView.setOnClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
+                    highlightSelectedOption(position, v);
                     itemClicked(position,v);
                 }
             });
         }
 
+        private void highlightSelectedOption(int position, View v) {
+            currentSelection = position;
+            int color = ContextCompat.getColor(v.getContext(), R.color.md_grey_200);
+            int length = titlelist.length();
+            for(int i = 0;i < length;i++)
```

Comment:
Why is for loop used here? It can be done using the position passed in as the parameter.